server/paths/refer.ts

```diff
@@ -7,17 +7,24 @@ const newReferralPeoplePathBase = offeringReferralPathBase.path('people')
 
 const referPathBase = path('/refer')
 
-const newReferralsPath = referPathBase.path('referrals/new')
+const referralsPath = referPathBase.path('referrals')
+
+const caseListPath = referralsPath.path('case-list')
+
+const newReferralsPath = referralsPath.path('new')
 const newReferralShowPath = newReferralsPath.path(':referralId')
 const newReferralProgrammeHistoryPath = newReferralShowPath.path('programme-history')
 const newReferralProgrammeHistoryShowPath = newReferralProgrammeHistoryPath.path(':courseParticipationId')
 const newReferralProgrammeHistoryProgrammePath = newReferralProgrammeHistoryShowPath.path('programme')
 const newReferralConfirmOasysPath = newReferralShowPath.path('confirm-oasys')
 const newReferralAdditionalInformationPath = newReferralShowPath.path('additional-information')
 
-const referralShowPathBase = referPathBase.path('referrals/:referralId')
+const referralShowPathBase = referralsPath.path(':referralId')
 
 export default {
+  caseList: {
+    show: caseListPath.path(':referralState'),
```

Comment:
Are we referring to the three referral status groupings as 'referral state'? I wonder if this might get confusing given how similar in meaning they are? Is `referralStatusGroup(ing)` any clearer (here and elsewhere)?

Reply:
I've gone for variations of referral status group.